internal_transfers/actions/src/utils.ts

```diff
@@ -20,13 +20,14 @@ export async function getTxDataFromHash(
     throw new Error(`invalid transaction hash ${txHash} - try again`);
   }
   const { from, hash, blockNumber, logs: readonlyLogs } = transaction;
-  const logs = readonlyLogs.map((log) => {
-    return {
-      ...log,
-      // had to Map it to make a copy of a readonly field.
-      topics: log.topics.map((value) => value),
-    };
-  });
+  const logs: EventLog[] = readonlyLogs.map(
+    ({ index, address, data, topics }) => ({
+      index,
+      address,
+      data,
+      topics: [...topics],
```

Comment:
I would keep the comment otherwise in the future people my look at this and remove the `[...topics]` part since it's redundant.

Reply:
Done!